ci: clarify OSS vs Enterprise plan for mirrord ci (#284)
metalbear-co/docs: docs/use-cases/mirrord-for-ci.md

tags:
  - beta
  - oss

>   - team

Reviewer: whatever is in oss is in team

![How mirrord for CI works](../.gitbook/assets/mirrord-ci-diagram.png)

{% hint style="info" %}

> mirrord ci works with the open-source version of mirrord, no plan required. Running it with the mirrord Operator requires the Enterprise plan.

Reviewer: ```suggestion
Running mirrord for CI with preview environments requires the Enterprise tier.
```

Author: @liron-sel I feel that adding preview environments to the mix is only adding to the confusion. It's actually ci + operator that requires the enterprise license, and not any other feature. Since we're clarifying it later in the doc - I'd suggest remove this hint altogether.

Reviewer: ok so just change to 'mirrord for CI' or `mirrord ci` (the command)
i prefer the first option

It's recommended that you create a Kubernetes [service account](https://kubernetes.io/docs/concepts/security/service-accounts/) for the CI runner.

> #### Using mirrord ci with the mirrord Operator

Reviewer: its not a customer perspective imo.
```suggestion
#### Using mirrord for CI for Enterprise Tier
```

{% hint style="info" %}
This section only applies to Enterprise plan users. If you're using the open-source version of mirrord, skip ahead to [Starting a mirrord CI session](#starting-a-mirrord-ci-session).
{% endhint %}

> When the mirrord Operator is installed in the cluster, running mirrord ci requires a CI API key. Generating the key requires an Enterprise plan.

Reviewer: ```suggestion
When the mirrord Operator is installed in the cluster, `mirrord ci` comands require a CI API key. Generating the key requires an Enterprise plan.
```

You can get the key by running the command:

```sh
mirrord ci api-key
```

Copy it and save it as the **secret** environment variable `MIRRORD_CI_API_KEY` in your CI.

> Sessions started with a CI API key don't count towards your seats, so CI runs won't consume them.

Reviewer: would be nice to see the reasoning here. why wouldnt it
